lego/apps/users/search_indexes.py

```diff
@@ -31,12 +32,12 @@ def get_autocomplete(self, instance):
 
     def autocomplete(self, query):
         return self.queryset.annotate(
-            fullname=Concat(F("first_name"), Value(" "), F("last_name"))
+            search=SearchVector("first_name", "last_name", "username")
         ).filter(
-            Q(first_name__istartswith=query)
-            | Q(last_name__istartswith=query)
-            | Q(username__istartswith=query)
-            | Q(fullname__istartswith=query)
+            search=SearchQuery(
+                ":* & ".join(query.split() + [""]).strip("& ").strip(),
+                search_type="raw",
+            )
```

Comment:
This should have some better sorting :thinking: I'm thinking just `created_at` maybe is good? or maybe membership count or something (if that's not too discriminating?)